xwiki-platform-core/xwiki-platform-web/src/main/webapp/resources/uicomponents/viewers/diff.css

```diff
@@ -27,7 +27,7 @@
   border: 1px solid $theme.borderColor;
   font-family: Consolas, "Liberation Mono", "Courier New", monospace;
   font-size: 12px;
-  overflow: auto;
+  overflow: visible;
```

Comment:
This doesn't feel right. Any reason for this change?

Reply:
The intent is to avoid having a visible scrollbar at the bottom on Edge.

Reply:
Sure, but doesn't this mean that the diff can overflow the container? We show the diff in a few places where the width is limited. Have you checked that there's no overflow?

Reply:
I've replaced this overflow: visible with a table size of 99.99%.
I'm not a big fan of this solution but it has the advantage to preserve the current visual aspect of the table while:
- displaying the dashed border of the four sides (otherwise, the right border was hidden).
- prevent the display of an horizontal scrollbar at the bottom of the table